### ISSUE TITLE
Proteus Circuit diagram

### ISSUE THREAD

**Issue by Keith012** [imamhossain94/accident-detection-prevention-and-an-emergency-solution#1]
Hey imamhossain94
DId the project simulation in proteus work?

**Comment by imamhossain94** [comment]
No, it doesn't work. Because the modules and the sensors are not connected properly with the Arduino board.

**Comment by Keith012** [comment]
I like the project btw, brilliant work there
So you only did the hardware prototype? No simulations

**Comment by imamhossain94** [comment]
Only hardware. I did this project for my friend's final year university project.

**Comment by imamhossain94** [comment]
It is not possible to do such types of projects with Arduino Uno. We should have to use Arduino Mega.

**Comment by Keith012** [comment]
I guess I'll have to change from using Uno to Mega in this specific task. But yeah, it makes sense since the code is rather large.

Thanks for your help!

**Comment by imamhossain94** [comment]
Welcome.

**Comment by Mr-Joshva** [comment]
Hi...
I'm tring to do your project. Can you help me with more documentation..?

**Comment by imammasleap** [comment]
> Hi... I'm tring to do your project. Can you help me with more documentation..?

This is not a complete project. We have failed to fit all the sensors because of using Arduino UNO. We should use the Arduino Mega microcontroller. Also, the Proteus circuit diagram won't work. But, you can try to implement it.

**Comment by Suaib65** [comment]
> > Hi... I'm tring to do your project. Can you help me with more documentation..?
> 
> This is not a complete project. We have failed to fit all the sensors because of using Arduino UNO. We should use the Arduino Mega microcontroller. Also, the Proteus circuit diagram won't work. But, you can try to implement it.

brother can I use a breadboard to connect all the sensors to the UNO?

**Comment by Suaib65** [comment]
I was thinking of doing this project as my final year project. Can I use a breadboard to connect all the sensors to my UNO instead of buying a MEGA?

**Comment by imamhossain94** [comment]
No it's not possible.

**Comment by Suaib65** [comment]
> No it's not possible.

Is it possible if I cut down on few sensors like PIR sensor?

**Comment by Suaib65** [comment]
Brother please give me a response, I really need some help on this

**Comment by imamhossain94** [comment]
You can use some sensors with UNO, but my suggestion is to simulate it on proteus before implementing them in real life.

**Comment by Suaib65** [comment]
> You can use some sensors with UNO, but my suggestion is to simulate it on proteus before implementing them in real life.

Ok thanks bro..
Can I get your LinkedIn to connect ?

**Comment by Suaib65** [comment]
heyy bro why have you used the hc-05 bluetooth module?
